_rules/target-size-enhanced-gi8qkf.md

```diff
@@ -0,0 +1,819 @@
+			z-index: 6;
+			position: absolute;
+			top: 0;
+			overflow-y: scroll;
+			height: 80px;
+			pointer-events: none;
+		}
+		.spacer {
+			height: 30px;
+		}
+	</style>
+</head>
+<body>
+	<div class="scroller">
+		<div class="cover bad"></div>
+		<div class="spacer"></div>
+	</div>
+
+	<button id="target" class="highlightable" onclick="alert('Hello')">
+		Say Hello
+	</button>
+	<div class="hlines good highlight"></div>
+	<div class="vlines good highlight"></div>
+</body>
+```
+
+#### Failed Example 8
+
+These radio buttons have their size modified by the author and are therefore not [User Agent controlled components][user agent controlled component]. Their [clickable area][] is too small.
+
+```html
+<style>
+	input[type='radio'] {
+		width: 20px;
+		height: 20px;
+	}
+</style>
+<fieldset>
+	<legend>Pick a color (required)</legend>
+	<label><input type="radio" name="color" value="blue" />Blue</label>
+	<label><input type="radio" name="color" value="yellow" />Yellow</label>
+</fieldset>
+```
+
+#### Failed Example 9
+
+The zoom buttons do not have [essential size][]; they are too small. The pin (red square) on this map has [essential size][].
+
+```html
+<style>
+	.map {
+		background-image: url('/test-assets/target-size/map-background.jpg');
+		width: 1250px;
+		height: 1250px;
+	}
+	.zoom {
+		position: absolute;
+		left: 1230px;
+		height: 20px;
+		width: 20px;
+		background-color: white;
+		border: 1px solid red;
+	}
+</style>
+
+Location of ACT rules headquarters:
+<div class="map"></div>
+<input
+	type="image"
+	class="zoom"
+	src="/test-assets/target-size/zoom-in.png"
+	style="top: 35px;"
+	alt="Zoom in"
+	onclick="alert('Zooming in.')"
+/>
+<input
+	type="image"
+	class="zoom"
+	src="/test-assets/target-size/zoom-out.png"
+	style="top: 56px;"
+	alt="Zoom out"
+	onclick="alert('Zooming out.')"
+/>
+```
+
+#### Failed Example 10
+
+The [clickable area][] of this button does not contain a 44×44px [horizontal rectangle][].
+
+```html
+<style>
+	#target {
+		width: 24px;
+		height: 24px;
+		border-radius: 0;
+		rotate: 45deg;
+	}
+</style>
+<button id="target" onclick="alert('Hello')">Hi</button>
+```
+
+#### Failed Example 11
+
+The [clickable area][] of this button does not contain a 44×44px [horizontal rectangle][].
+
+```html
+<style>
+	#target {
+		width: 40px;
+		height: 40px;
+		border-radius: 30%;
+	}
+</style>
+<button id="target" onclick="alert('hello')">Hello</button>
+```
+
+#### Failed Example 12
+
+The [clickable area][] of this button only contains a 25×45px [horizontal rectangle][].
+
+```html
+<head>
+	<title>Failed Example</title>
+	<link rel="stylesheet" href="/test-assets/target-size/shared-styles.css" />
+	<style>
+		#target {
+			height: 50px;
+			width: 40px;
+			text-align: center;
+			clip-path: polygon(10px 0px, 10px 45px, 35px 45px, 35px 0px);
+			background-color: #0074d9;
+		}
+	</style>
+</head>
+<body>
+	<div id="target" role="button" onclick="alert('Hello')">
+		Hi
+	</div>
+</body>
+```
+
+### Inapplicable
```

Comment:
Shouldn't we have one with a clickable area that is not a semantic widget?

Reply:
🤔 Wouldn't that be bad for 1.3.1?
a `<span onclick="alert('Hello')">Say hello</span>` would fail 1.3.1. We can have one (with associated caveats), but I'm afraid it would cause problems for tools that do not have the same separation as ACT rules and would (rightfully) fail it.